holidays/countries/australia.py

```diff
@@ -50,12 +50,20 @@ class Australia(ObservedHolidayBase, ChristianHolidays, InternationalHolidays, S
     """
 
     country = "AU"
-    supported_categories = (BANK, HALF_DAY, PUBLIC)
     default_language = "en_AU"
     # %s (observed).
     observed_label = tr("%s (observed)")
-    supported_languages = ("en_AU", "en_US", "th")
-    subdivisions = ("ACT", "NSW", "NT", "QLD", "SA", "TAS", "VIC", "WA")
+    start_year = 1801
+    subdivisions = (
+        "ACT",  # Australian Capital Territory.
+        "NSW",  # New South Wales.
+        "NT",  # Northern Territory.
+        "QLD",  # Queensland.
+        "SA",  # South Australia.
+        "TAS",  # Tasmania.
+        "VIC",  # Victoria.
+        "WA",  # Western Australia.
+    )
```

Comment:
@KJhellico @PPsyrius what do you think about this approach?
```suggestion
    # fmt: off
    subdivisions = (
        "ACT",  # Australian Capital Territory.
        "NSW",  # New South Wales.
        "NT",   # Northern Territory.
        "QLD",  # Queensland.
        "SA",   # South Australia.
        "TAS",  # Tasmania.
        "VIC",  # Victoria.
        "WA",   # Western Australia.
    )
    # fmt: on
```

Reply:
This seems better for code readability